react/ListPrice.tsx

```diff
@@ -14,8 +14,12 @@ const CSS_HANDLES = [
   'taxPercentage',
 ] as const
 
-const ListPrice: StorefrontFC<BasicPriceProps> = props => {
-  const { message, markers } = props
+interface ListPriceProps {
```

Comment:
Since you only want to change the value of the savings, are the changes to this file really necessary?

Reply:
@iaronaraujo Yes, because I think not everyone does use the spot price, so this new prop will grant some security to not break the other stores.

Reply:
Hey @gustavopvasconcellos I think that instead of creating a new component we could create a new interpolable variable with the "list spot price"! 
But before we proceed with this, do you have an example of how this could be used? I don't know if I've ever seen a store using the spot price discount on the list price, so if you have an example it would help me understand better this demand so we can think about the best solution to solve the problem!

Reply:
@iaronaraujo Of course, we have this product above and there is a spotPrice, but the ListPrice and Price are the same and the product-list-price doesn't show, either the discounts percentage.

You can found in this page: https://carrefourbr.myvtex.com/eletroportateis/bebedouro-e-purificador-de-agua/cor_Branco
If you can't, try the next pages please, this store is changing everytime xD

![image](https://user-images.githubusercontent.com/49173685/91451867-4756f700-e854-11ea-9c8e-27958c11f972.png)

Reply:
@gustavopvasconcellos 
I see! On this case, since we are only interested in showing the discount, I think we won't need to change the `ListPrice`component for now and if you want to show the previous price, you can do so with the `SellingPrice` component since it is the same price on this store! You can then add the same `strike` effect on it by using CSS!

Reply:
Actually, I think that this could lead to some problems in the future. What do you think about adding a prop `shouldAlwaysShow` on `ListPrice` so that you can *always* show the list price? I like this idea better than having some logic to decide if the list price will be rendered or not because this way we can add this prop to the site-editor so the client can choose when to show it!